ui-tests/src/test/java/org/openmrs/reference/RetireVisitTypeTest.java

```diff
@@ -0,0 +1,51 @@
+package org.openmrs.reference;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.VisitTypeListPage;
+import org.openmrs.reference.page.VisitTypePage;
+import org.openmrs.uitestframework.test.RestClient;
+import org.openmrs.uitestframework.test.TestData;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ */
+public class RetireVisitTypeTest extends ReferenceApplicationTestBase{
+
+    public static final String RETIRE_REASON = "Retire reason";
+    private String visitTypeName;
+    private String visitTypeDesc;
+
+    private String visitTypeUuid;
```

Comment:
I have change this a little bit but not exactly like you wanted. Please check it and say what you think.